scripts/test/shared.py

```diff
@@ -225,10 +225,9 @@ def has_shell_timeout():
 
 # Default options to pass to v8. These enable all features.
 V8_OPTS = [
+    '--wasm-staging',
```

Comment:
- `--wasm-staging` [includes](https://github.com/v8/v8/blob/ffb4f2b77fd3abe3d7450312aa3c670776d9502d/src/wasm/wasm-feature-flags.h#L32-L71) `mv` and `threads`, so they can be removed
- Bulk memory is shipped and [enabled by default](https://github.com/v8/v8/blob/ffb4f2b77fd3abe3d7450312aa3c670776d9502d/src/wasm/wasm-feature-flags.h#L73-L82), so we can remove that (and the feature flag is scheduled to be removed)
- There is one more flag that is not enabled by default or `--wasm-staging`: [`--experimental-wasm-compilation-hints`](https://github.com/v8/v8/blob/ffb4f2b77fd3abe3d7450312aa3c670776d9502d/src/wasm/wasm-feature-flags.h#L28-L30). Binaryen doesn't use this for anything for now but maybe it'd be better to include this for completeness..?

Reply:
Thanks! Updated.